sphinx/source/developers/demo.rst

```diff
@@ -5,26 +5,26 @@ This document explains how the Python testing infrastructure is used to run a co
 
 The script ``tests/e2e_scenarios.py`` reads a test scenario from json. This json file can specify which app the network should run, how many nodes it should create, and the list of transactions to run. ``tests/simple_logging_scenario.json`` is an example scenario file showing the expected format:
 
-.. literalinclude:: ../../tests/simple_logging_scenario.json
+.. literalinclude:: ../../../tests/simple_logging_scenario.json
     :language: json
 
-To execute this scenario first follow the instructions in :ref:`getting_started` and then, from the ``build`` directory, run:
+To execute this scenario first follow the instructions in :ref:`Start Here` and then, from the ``build`` directory, run:
```

Comment:
I think we now assume that readers have at least read (and hopefully run) all of the "Start Here" steps before they see this, so we don't need to call it out here.

Reply:
And a minor typo in the last sentence in this file that I can't comment on directly - should be "alive, rather than", not "alive rather, than".